src/OddJobs/ConfigBuilder.hs

```diff
@@ -63,14 +63,15 @@ mkConfig logger tname dbpool ccControl jrunner configOverridesFn =
             , cfgDbPool = dbpool
             , cfgOnJobStart = (const $ pure ())
             , cfgDefaultMaxAttempts = 10
-            , cfgTableName = tname
+            , cfgTableNames = simpleTableNames tname
```

Comment:
This allows backwards compatibility, assuming that for simple use cases, users don't plan on using resources and don't care about controlling every table name.